doc/Makefile

```
@@ -19,10 +19,11 @@ help:
 	@echo "To run linkcheck, use 'make <target> linkcheck=true'"
 	@echo "To run doctests, use 'make <target> doctest=true'"
 	@echo "To fix outdated doctests, use 'make <target> doctest=fix'"
+	@echo "To run doctests using Revise, use 'make <target> doctest=true revise=true'"
```

That is technically accurate, but not very helpful. To the non-expert, it remains unclear what that does. I am hoping (also based on what @simeonschaub told me on Slack) that it means: "if you have docstring changes that are not in the sysimage, we will take those into account when running the doctests". That would be awesome.

Assuming I got this right, how about this:
```suggestion
	@echo "To run doctests using Revise (to test changes w/o regenerating the sysimage), use 'make <target> doctest=true revise=true'"
```